src/Shell/Task/TemplateTask.php

```diff
@@ -140,6 +147,9 @@ public function main($name = null, $template = null, $action = null)
         if (!$action) {
             $action = $this->template;
         }
+        if (!empty($this->params['index-columns'])) {
+            $this->maxIndexColumns = (int)$this->params['index-columns'];
```

Comment:
What does the property do that the param index does not? Seems like you could just use the param value.